models/articleModel.js

```diff
@@ -9,4 +9,18 @@ exports.getArticleById = (article_id) => {
         }
         return rows[0];
     });
-};
\ No newline at end of file
+};
+exports.fetchAllArticles = () => {
+    return db
+      .query(`
+        SELECT articles.author,  articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, 
+        COUNT(comments.comment_id) AS comment_count
+        FROM articles
+        LEFT JOIN comments ON comments.article_id = articles.article_id
+        GROUP BY articles.article_id
+        ORDER BY articles.created_at DESC;
+      `)
+      .then(({ rows }) => {
```

Comment:
Really well done! It might've taken you a while to figure out, but you smashed it :)